vtex-apps/checkout-cart
# Add number of unavailable and total items to shipping-calculator

#### What problem is this solving?

We need to show contextualized warnings about the availability of products to be delivered to a given address. To do that, `shipping-calculator` needs to know the amount of unavailable items as well as the total amount of items added to cart, since there's two types of availability warnings as it's shown below. This PR adds it.



#### How should this be manually tested?

- Proceed to [Workspace](https://deliveryfeedback--checkoutio.myvtex.com/cart/add?sku=308&sku=288)
- Verify that a warning (image 1) it's being shown.
- Remove the available item.
- Verify that another type of warning (image 2) is being shown now.

#### Screenshots or example usage

**Image 1**

![image](https://user-images.githubusercontent.com/12574426/70931866-2d072100-2017-11ea-95e7-5a89cc33a85b.png)

**Image 2**

![image](https://user-images.githubusercontent.com/12574426/70931909-43ad7800-2017-11ea-8f4b-450eb093af08.png)


#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable).](https://app.clubhouse.io/vtex/story/27145/implementar-tratamento-de-erros-para-delivery-options)

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
